src/wp-includes/script-modules.php

```diff
@@ -112,3 +112,14 @@ function wp_enqueue_script_module( string $id, string $src = '', array $deps = a
 function wp_dequeue_script_module( string $id ) {
 	wp_script_modules()->dequeue( $id );
 }
+
+/**
+ * Unregisters the script module.
```

Comment:
```suggestion
 * Deregisters the script module.
```

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/6061/commits/11d2ea7ecbec9b63f1d20fbda5007397b3153b95